app/policies/leave_policy.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class LeavePolicy < ApplicationPolicy
+  def index?
+    user_owner_role? || user_admin_role?
+  end
+
+  def create?
+    user_owner_role? || user_admin_role?
+  end
```

Comment:
Can we move common conditions like this as a separate method?

```
def has_owner_or_admin_role?
    user_owner_role? || user_admin_role?
end
```
```suggestion
  def index?
    has_owner_or_admin_role?
  end

  def create?
    has_owner_or_admin_role?
  end
```